### DISCUSSION TITLE
Verify my Phone Number? hell no

### DISCUSSION THREAD

**Discussion by Hawkmoon9** [chromiecraft/chromiecraft#4917] [General]
Convinced 4 of my friends to leave Stormforge to try ChromieCraft. We started client download, joined the Discord channel and then... wtf? you want our cell phone numbers to read or post in Discord? Why? I have never seen this from any server - ever.  You just lost 5 new players. Just asking you to reconsider this - could be one of the reasons why the server has so few players? back to Stormforge. best of luck

**Comment by BeefSteaker** [comment]
The Verification is discord side not chromie side. 
The server requires a Verified account to avoid spam and bots, and unfortunately the only way discord verifies accounts is via phone

https://support.discord.com/hc/en-us/articles/360000961212-Phone-Verification

**Reply by Hawkmoon9** [reply]
> The Verification is discord side not chromie side. The server requires a Verified account to avoid spam and bots, and unfortunately the only way discord verifies accounts is via phone
> 
> https://support.discord.com/hc/en-us/articles/360000961212-Phone-Verification

I am currently an active member of 10 discord servers. My account is fully verified. Not myself or any of my friends have ever encountered this from any server - only ChromieCraft. So this is something on ChromieCraft's end.

**Reply by r-o-b-o-t-o** [reply]
As Beef explained, this is a Discord setting we chose to use in order to combat bot accounts and spammers.

While it is indeed a choice made on our end, we **do not have access** to your phone number if you choose to use it to verify your account. In the same vein, we cannot access any other data as your IP address used to connect to Discord or the email address linked to your Discord account. This is all stored on Discord's servers and we have no access to this data.

**Reply by Hawkmoon9** [reply]
Ya, We dont want to give our cell phone numbers to anyone - not to a WoW private server or Discord. But ok if you dont want to just ban spam accounts manually i guess thats your choice. Just sayin this is gonna scare away real human players. I sincerely hope your server thrives - we love the idea of a slow progression, challenging WoW server. best of luck

**Reply by 55Honey** [reply]
> My account is fully verified

Sorry but it can not possibly be fully verified, because the highest level of security which discord.ofders, is the phone verification.

Roboto and Beef have already said it: You do not verify your account against the ChromieCraft server, but **only with discord** itself. No data from that verification, certainly NOT your phone number would become accessible to the ChromieCraft server owner/admins.

I wish you best of luck at whatever server you decide to play on. And i also hope they have some other kind of decent bot/spam protection to prevent malicious accounts to send DMs to every member of their discord.

**Reply by 55Honey** [reply]
> But ok if you dont want to just ban spam accounts manually

Oh don't worry, we certainly ban them. But i prefer our community to be save from even being bothered by them. Because if they cant even join, they don't stand a chance to post potentially malicious links.

**Reply by Hawkmoon9** [reply]
Understood. As one of our friends pointed out, we can just use this forum and ingame chat for questions, feedback, or seeking a guild. So i guess Discord isnt 100% necessary. We may have had a hyper reaction to this but we've never seen that before and all defend our cyber-privacy as much as we can. We'll hopefully give the server a try soon. thanks for your time.

**Reply by 55Honey** [reply]
The server discord is indeed nowhere near mandatory. Welcome to ChromieCraft!

**Reply by Amrx2** [reply]
> Understood. As one of our friends pointed out, we can just use this forum and ingame chat for questions, feedback, or seeking a guild. So i guess Discord isnt 100% necessary. We may have had a hyper reaction to this but we've never seen that before and all defend our cyber-privacy as much as we can. We'll hopefully give the server a try soon. thanks for your time.

Hi Hawkmoon9, firstly I hope you all enjoy your stay on Chromiecraft! We take Data Protection as serious as anyone as well, if you are ever concerned about anything you can have a read over our privacy policy about what we do collect on our website - https://www.chromiecraft.com/en/privacy-policy-2/.

**Reply by Daedalus007** [reply]
Server verification is server-side, not Discord-side.
https://support.discord.com/hc/en-us/articles/216679607--Verification-Levels

It is difficult to tell people to file tickets on Discord while also requiring a phone number to verify.  People don't want to give such info to the Discord company for any reason and that's perfectly reasonable.
Also many people with older accounts (like myself) have been Discord and/or Game Account banned for no stated reason.
If you all want to keep the current system, have the ban system show this information in the account settings after logging in on the website.  Otherwise it is terrible for people to need to resort to posting publicly on Github for a response.

Your website still has issues with allowing people to login; if a player has 2FA enabled they can't even login via the main ChromieCraft website unless they happen to know the exact login URL to use (which does work properly).
Login URL: https://www.chromiecraft.com/wp-login.php

But instead of just clicking on the login button (which has no ALT text to know it is even a login button) and redirecting to that page, it puts players in an endless 'no 2FA code entered' loop.

I'd like to add my voice to the mix and request the removal of phone verification on the Chromiecraft Discord and the addition of ban tickets/appeals/etc into the website account system for those who are banned in-game and from the Discord itself.

**Reply by shaszun** [reply]
Hi, I hope this won't burst your bubble, but if you want to protect your privacy online you need to stop using smart phones or any other smart tech. If it's free, you are paying the price with your data. Examples - 9gag app is very invasive, smart washing machine using 2gb of data per day. Start getting used to it, no way to avoid it.

**Comment by balleny** [comment]
Never verified my discord account and can post and access all channels but offtopic one (which gives me the popup to verify)

**Reply by Hawkmoon9** [reply]
![Screenshot 2023-02-03 155647](https://user-images.githubusercontent.com/124067507/216709222-62f3c9a9-ef81-4591-99d6-b2d348a45a97.jpg)

well I cant, and neither can my friends.

**Reply by Daedalus007** [reply]
Hawkmoon is correct; new accounts that join cannot chat anywhere without mandatory phone verification (set by the server, not by Discord).  Even if Discord is the one who gets the phone number, many people are not ok with giving IRL info to some random tech company.

**Reply by balleny** [reply]
indeed. as much as spam protection has a reason, I would also never give out my phone number to a company like discord.
there should be better solutions in place, especially given that a lot of the communication stopped taking place via the homepage

**Comment by 55Honey** [comment]
If there was an alternative as effective as the phone verification, it will be considered. I haven't seen a usable suggestion for a replacement yet.
